i18n/venue/index.i18n.js

```diff
@@ -8,9 +8,8 @@ export default genI18nMessages({
                 'PyCon Taiwan 2023 will be held at the Humanities and Social Science Building, Academia Sinica, with a focus on the latest technology and best practices in Python. The open-source community will bring high-quality speeches and workshops, providing valuable learning and networking opportunities for both novice and experienced Python developers.',
```

Comment:
Might have to inquire the description of 2024 from team venue.

Reply:
This would be updated at https://github.com/pycontw/pycontw-frontend/pull/533/commits/23da2ac5b2d14db4f9d0850f5227fc87cf190923
plz check.

Reply:
I think we should minimize the difference between Zh/En. 

How about that?
'PyCon Taiwan 2024 will be held at the National Science and Technology Museum (South Complex), with a focus on the latest technology and best practices in Python. The open-source community will bring high-quality speeches and workshops, providing valuable learning and networking opportunities for both novice and experienced Python developers.'

Reply:
updated thanks